ui/app/AppLayouts/Onboarding2/UseRecoveryPhraseFlow.qml

```diff
@@ -24,34 +22,29 @@ SQUtils.QObject {
     signal setPasswordRequested(string password)
     signal finished
 
-    function init(type = UseRecoveryPhraseFlow.Type.NewProfile) {
-        let title = ""
-
-        if (type === UseRecoveryPhraseFlow.Type.NewProfile)
-            title = qsTr("Create profile using a recovery phrase")
-        else if (type === UseRecoveryPhraseFlow.Type.KeycardRecovery)
-            title = qsTr("Enter recovery phrase of lost Keycard")
-        else if (type === UseRecoveryPhraseFlow.Type.Login)
-            title = qsTr("Log in with your Status recovery phrase")
-
-        root.stackView.push(seedphrasePage, { title })
-    }
+    initialItem: SeedphrasePage {
+        isSeedPhraseValid: root.isSeedPhraseValid
 
-    Component {
-        id: seedphrasePage
+        title: {
+            if (root.type === UseRecoveryPhraseFlow.Type.NewProfile)
+                return qsTr("Create profile using a recovery phrase")
+            else if (root.type === UseRecoveryPhraseFlow.Type.KeycardRecovery)
+                return qsTr("Enter recovery phrase of lost Keycard")
+            else if (root.type === UseRecoveryPhraseFlow.Type.Login)
+                return qsTr("Log in with your Status recovery phrase")
```

Comment:
Why not `switch/case`? 🤔 
```suggestion
            switch (root.type) {
            case UseRecoveryPhraseFlow.Type.NewProfile:
                return qsTr("Create profile using a recovery phrase")
            case UseRecoveryPhraseFlow.Type.KeycardRecovery:
                return qsTr("Enter recovery phrase of lost Keycard")
            case UseRecoveryPhraseFlow.Type.Login:
                return qsTr("Log in with your Status recovery phrase")
            }
```

Reply:
6 lines vs 8 lines, I would keep as at is, no big difference in the end I think :)

Reply:
The difference is the readability 🙂 

With switch/case the intentions are obvious, and you only reference `root.type` once. 
While with `else if`, I _suppose_ that the intention is switch/case, but in fact I have to read and check every condition to make sure what's happening here.

> 6 lines vs 8 lines

I think it's not always just about making the code shorter.
But if you want numbers, then it's 421 characters vs 417 characters haha 😄

Reply:
If you want to know the truth, it's about how qtc formats switch statement. I truly hate lack of indentation inside brackets there :rofl: